bin/run-lint

```diff
@@ -1,21 +1,42 @@
 #!/usr/bin/env node
-const child_process = require( 'child_process' );
+
 const path = require( 'path' );
-var args = [ '--cache', '--quiet', '--ext=.js',  '--ext=.jsx' ];
+const child_process = require( 'child_process' );
+const stylish = require( 'eslint/lib/formatters/stylish' );
+const exitCodeFromEslint = require( './eslint-formatters-calypso/src/lib/exit-code' );
+
+var args = [ '--cache', '--quiet', '--ext=.js', '--ext=.jsx' ];
+
+const argsBranchName = [ 'rev-parse', '--abbrev-ref', 'HEAD' ];
+const branchName = child_process.spawnSync( 'git', argsBranchName ).stdout.toString().trim();
+
+if ( branchName === 'master' ) {
```

Comment:
Thanks for the suggestion! At the moment, what I have done is to create a new formatter at `eslint-formatters-calypso`: it is only a wrapper for deciding which formatter to return depending on the git branch. I would need to check how different editors call `eslint`, but I guess passing a formatter like in `eslint -f eslines .` should be possible. However it may be, moving the logic inside `eslint-formatters-calypso` is a good idea.

As an aside: `eslines` has been rebranded to a more meaningful name: `eslint-formatters-calypso`, as its responsibilities have grown and changed. `eslines` is now the name of the _smart_ and wrapper formatter inside that package.